js-jasmine/src/backstage-passes.js

```diff
@@ -0,0 +1,7 @@
+const Item = require('./item');
+
+class BackstagePass {
+    constructor(sellIn, quality) {
+        super("Backstage passes", sellIn, quality);
```

Comment:
Really concise, I like the use of super, haven't used it before! Interested in how it works